test/library/run-options.test.js

```diff
@@ -129,7 +129,7 @@ describe('Newman run options', function () {
                 bail: ['folder', 'failure']
             }, function (err) {
                 expect(err).to.be.ok();
-                expect(err.message).to.be('Unable to find a folder or request: invalidName');
```

Comment:
?

Reply:
@deepakpathania I'm guessing the place where this error was thrown must be changed from `new Error('error' + options.entrypoint)` to `new Error('error' + options.entrypoint && options.entrypoint.execute)`

Reply:
@kamalaknn Yeah, it would require the mentioned change [here](https://github.com/postmanlabs/postman-runtime/blob/develop/lib/runner/index.js#L92) in the runtime library, simply replacing the `to.be` with `to.contain` felt simpler to me and fixed the issue. 
@kunagpal Should I send a PR to runtime with the change?

Reply:
Fixed in [this](https://github.com/postmanlabs/postman-runtime/pull/500) PR. Please review, once merged, can revert this to `to.be`, if required.

Reply:
The PR on runtime looks good, left one comment.

Like @kunagpal has mentioned, you would want to add a test cases for 

1. Entry point being an invalid type (string)
2. Entry point passed in the correct format, with missing folder.

I think there are no integration tests that assert on collections with entrypoint to a folder. That's why we've seen this bug ship out. Can you add an integration test for `--folder` option?

Reply:
@deepakpathania Now that runtime `v7.1.2` is out, this can be updated.